RecoLocalTracker/SubCollectionProducers/src/TrackClusterRemover.cc

```diff
@@ -51,8 +51,8 @@ namespace {
     const TrackCollectionTokens trajectories_;
     edm::EDGetTokenT<QualityMaskCollection> srcQuals;
     
-    const edm::EDGetTokenT<edmNew::DetSetVector<SiPixelCluster> > pixelClusters_;
-    const edm::EDGetTokenT<edmNew::DetSetVector<SiStripCluster> > stripClusters_;
+    edm::EDGetTokenT<edmNew::DetSetVector<SiPixelCluster> > pixelClusters_;
```

Comment:
If those stay const, compilation fails with this error:
data/user/jschulte/CMSSW_9_2_4/src/RecoLocalTracker/SubCollectionProducers/src/TrackClusterRemover.cc:98:19: error: passing 'const edm::EDGetTokenT<edmNew::DetSetVector<SiPixelCluster> >' as 'this' argument discards qualifiers [-fpermissive]
    pixelClusters_ = consumes<edmNew::DetSetVector<SiPixelCluster> >(pixelClusters);